.github/workflows/prerelease.yml

```
@@ -79,12 +79,26 @@ jobs:
           aws-region: ${{ env.BUILD_REGION }}
           role-to-assume: ${{ secrets.BUILD_ROLE_TO_ASSUME_STAGING }}
           role-duration-seconds: 3600
+      # When called by Packer, Ansible will find /usr/bin/python3 and
+      # use it; therefore, we must ensure that /usr/bin/python3 points
+      # to the version of python that we installed.  This can hose
+      # other tasks that are expecting to find the system python at
+      # that location, though, so we undo this change after running
+      # packer.
```

I think we should flesh this out to mention that we care about the version of Python installed by `actions/setup-python`, as it is a bit ambiguous the way it's written.

I also had a couple of capitalization thoughts:
```suggestion
      # When called by Packer, Ansible will find /usr/bin/python3 and
      # use it; therefore, we must ensure that /usr/bin/python3 points
      # to the version of Python that we installed.  This can hose
      # other tasks that are expecting to find the system Python at
      # that location, though, so we undo this change after running
      # packer.
```

---

See commits fa6c1c1 and 51b729c.

---

Thanks!